docs/dialect.md

```diff
@@ -133,3 +133,40 @@ The <span style="font-family: courier new">argument</span> element is an opt
 Allowable types: none, bool, real, int, str, datetime, date, localstr, null, error, any, tuple, spatial, localreal, localint.  
 Allowable date parts: year, quarter, month, dayofyear, day, weekday, week, hour, minute, second. 
 
+### Join Support:
+**Format is distinct** <br/>
+ `<format-is-distinct>` Defines a strategy for determining whether two values are distinct.  <br/>
+ The value for this property can be: 
+
+  - __NoNullCheck__
+      Does not check that the value of LHS and RHS is null. Formula: `(lhs [!]= rhs)`
+  - __Keyword__
+      Uses `DISTINCT` keyword for comparision. Formula: `(lhs IS [NOT ]DISTINCT FROM rhs)`.
+  - __Operator__
+    Uses `<=>` for comparions. Formula:`([NOT] (lhs <=> rhs))`
+
+  By default, format is distinct checks that the value for LHS and RHS is not null.   <br/>
+  Formula :`((lhs [!]= rhs) OR[AND] (lhs IS [NOT] NULL AND[OR] rhs IS [NOT] NULL))`
```

Comment:
I think the Formula should be also listed as a possible value for this property, after NoNullCheck, Keyword, and Operator.

Reply:
+1

Reply:
Good catch made the changes.